postgrest.cabal

```diff
@@ -88,15 +92,12 @@ library
                     , wai-extra                 >= 3.0.19 && < 3.2
                     , wai-logger                >= 2.3.2
                     , wai-middleware-static     >= 0.8.1 && < 0.10
-  default-language:   Haskell2010
-  default-extensions: OverloadedStrings
-                      QuasiQuotes
-                      NoImplicitPrelude
-  if flag(FailOnWarn)
-    ghc-options: -O2 -Werror -Wall -fwarn-identities
+  if flag(dev)
+    ghc-options: -O0 -Werror -Wall -fwarn-identities
```

Comment:
@steve-chavez Added -O0 for development and it feels faster. Didn't time it, though.